sdk/python/feast/type_map.py

```diff
@@ -89,6 +89,10 @@ def feast_value_type_to_pandas_type(value_type: ValueType) -> Any:
         ValueType.BYTES: "bytes",
         ValueType.BOOL: "bool",
         ValueType.UNIX_TIMESTAMP: "datetime",
+        ValueType.DOUBLE_LIST: "object",
+        ValueType.FLOAT_LIST: "object",
+        ValueType.INT32_LIST: "object",
+        ValueType.INT64_LIST: "object",
```

Comment:
Might it be easier to have a:
```python
# All list types are `object` dtypes in Pandas
if "_LIST" in value_type.name:
    return 'object'
```
since any list type will be an object in Pandas (I also wonder if Feast can move away from handling Pandas type conversions and let PyArrow do it? I haven't given that much thought though.)

Reply:
For example you've missed:
* `ValueType.BOOL_LIST`
* `ValueType.BYTES_LIST`
* `ValueType.UNIX_TIMESTAMP_LIST`
* `ValueType.STRING_LIST`

Reply:
@judahrand Thank you for the excellent suggestion. I've updated this PR and I hope you don't mind that I used `value_type.name.endswith("_LIST")` instead.

Reply:
Even better!